crates/bevy_app/src/app_builder.rs

```diff
@@ -215,6 +276,25 @@ impl AppBuilder {
     }
 
     /// Inserts a resource to the current [App] and overwrites any resource previously added of the same type.
+    ///
+    /// A resource in bevy represents globally unique data. The resources must be added to bevy application
+    /// before using them. This happens with [`AppBuilder::insert_resource`].
+    ///
+    /// For adding a main-thread only accessible resource, see [`AppBuilder::insert_thread_local_resource`].
```

Comment:
This will be incorrect as of Bevy 0.5. These have been renamed to `NonSend` resources.

Reply:
(I removed the line for now)